src/main/java/com/example/CarDealership/CarDealershipApplication.java

```diff
@@ -0,0 +1,42 @@
+package com.example.CarDealership;
+
+import com.example.CarDealership.config.DataSourceProperties;
+import com.example.CarDealership.models.Dealership;
+import com.example.CarDealership.models.Vehicle;
+import com.example.CarDealership.repositories.DealershipRepository;
+import com.example.CarDealership.repositories.VehicleRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import javax.xml.crypto.Data;
+import java.util.List;
+
+@SpringBootApplication
+public class CarDealershipApplication implements CommandLineRunner {
+
+	@Autowired
+	private DealershipRepository dealershipRepository;
+
+	@Autowired
+	private DataSourceProperties dataSourceProperties;
+    @Autowired
+    private VehicleRepository vehicleRepository;
+
+
+	public static void main(String[] args) {
+		SpringApplication.run(CarDealershipApplication.class, args);
+	}
+
+	@Override
+	public void run(String... args) throws Exception{
+//		List<Dealership> dealershipList = dealershipRepository.getAllDealerships();
```

Comment:
This is fine for testing, but we want to ask the user how they want to filter their data. Optionally, you could ask them what dealership they'd like to see vehicles for, and then filter by that.